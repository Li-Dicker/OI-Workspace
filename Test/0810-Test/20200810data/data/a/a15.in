10
196
68905 99676 12170 58410 35076 87013 57431 2051 65390 25301 50683 82750 85672 63393 92610 16370 43453 91372 53114 1842 26752 25102 2038 97863 77391 85214 89794 35244 83510 93228 58603 31256 97696 25968 6942 41252 8308 85407 59393 61871 31310 88976 11514 1760 86449 52666 50782 73655 35102 96313 99590 9455 99089 16298 42631 50400 32747 88187 24381 14845 30676 39197 38411 19361 60029 6017 39803 95864 79832 19816 13273 19209 57123 14157 11342 30876 22195 49292 78627 71345 51726 14075 65441 92698 80125 2041 56859 20157 54733 43428 86121 69514 69612 43222 49241 32035 28173 83504 97178 52136 30460 26096 79600 68406 6182 32249 15879 86030 92537 45761 22695 93196 9934 31002 79738 58570 98395 91915 44513 7986 37545 51014 91803 59490 80413 37059 83667 18328 23554 49587 80187 59712 24649 90940 38650 47913 13646 85972 86243 68425 90821 11412 16522 65354 3203 49811 19030 97680 54242 15274 97447 62218 3572 35833 69539 26958 20057 6681 81722 86570 18439 16862 6344 14035 43214 48518 16501 84856 10099 77323 58738 35182 77150 7650 41109 33733 44764 53466 74562 48113 10667 83386 81003 49243 50389 68557 34425 64944 35873 49396 35981 21940 32574 7577 80149 30286 
26612 40373 18941 42148 43251 28103 17999 17852 1099 48425 48679 36417 5796 37426 4056 14973 14971 37406 46349 49022 5573 35202 47260 34103 12561 25024 31878 9347 20952 18441 40022 18901 44545 14141 25646 39161 2465 32037 42074 12999 14790 41652 17819 22878 40776 13874 12590 38194 40719 2351 23458 8344 1101 48724 47741 7650 49876 25765 31224 31915 45719 46437 25183 19581 18519 3461 14671 48888 48326 11612 30288 6658 47843 13804 25095 11370 43118 11934 2989 20119 29154 45111 46267 35204 35432 29455 22977 48003 20088 41841 1393 27552 46896 2408 29737 7239 38023 20104 9321 8591 3739 29283 25257 614 31478 48365 20010 15354 16465 21952 15784 32495 48412 8131 37358 3445 43222 32944 13161 46919 33067 35663 47400 27650 11591 28822 45236 18484 21890 5795 109 41397 666 22639 38217 20367 31076 28791 33110 40386 42391 12255 11876 43976 16934 41769 26941 43403 5705 27131 38162 28901 2101 14853 485 8811 25518 35677 30164 38481 12723 31937 14825 1798 10318 24485 46247 9767 30974 27519 26315 34390 32468 2841 42843 21819 9091 24306 20365 43038 15142 329 22634 35707 5992 15291 32899 33775 39902 45088 911 17952 34173 975 20564 38087 
200
54648 17073 88846 52653 96272 88767 41165 36471 90710 73420 58489 28701 16799 73605 69396 54027 72268 695 3154 27948 6186 74742 64466 38945 82301 71330 53558 46548 52417 44830 88589 57476 26260 81760 38993 64637 54741 51509 47704 83546 78586 45218 95274 78558 42482 74261 91758 5766 4877 74378 28811 76838 71184 46820 50339 54669 90808 72417 85084 43032 96833 98649 17025 29913 47413 19081 34620 59344 73651 80434 99261 8964 50148 27285 19546 40625 71273 84595 11120 44948 80316 24077 85846 79165 59008 222 90441 47587 43872 72812 95420 6400 38181 92350 20955 70920 43230 76048 51620 47838 81872 40991 18354 43913 37979 36263 16370 61135 93237 72474 21336 6838 26826 31029 65099 49116 72244 67374 79343 50307 51188 62906 12276 94462 12549 3660 30137 58409 14004 5942 94087 19716 16194 41036 2696 30878 51546 31214 88867 42219 92165 18764 93126 41400 64715 1278 73789 27369 88662 5529 22951 87502 59806 78870 46983 45251 77460 51048 94081 48857 57150 17949 10203 22173 41617 77105 73128 68263 49443 28798 25475 61198 63166 89636 49731 47755 22224 81610 6135 93662 46991 50507 51983 64072 76043 40483 94465 47646 86 15619 75310 51263 10788 9917 64558 46502 27716 25964 29036 1930 
41829 33632 24892 40812 33660 40627 43860 39843 10499 21011 10996 21812 10426 10618 32307 1262 13727 18347 13948 32149 25741 46290 3555 49929 24042 19294 23822 29940 20033 5181 35141 8371 30520 32862 8613 49037 32074 26052 15917 34531 19514 17217 19481 18983 43733 35107 37563 12351 10157 12115 48952 12661 26570 14259 45566 26328 8835 49564 12689 13787 40084 41123 22307 42191 13344 7966 41559 1806 2762 34229 31257 22105 26664 16614 25057 187 10729 40181 49683 40427 28341 25358 48229 38734 15590 10989 46595 16943 8344 31091 30609 41966 37653 24363 41383 37450 18651 45784 47281 31859 17568 37861 22411 18333 634 26028 49216 21546 131 23022 33932 17848 39445 2121 43957 29963 37062 31928 47803 44474 23023 21730 21816 43064 15991 42060 400 35135 38650 19338 17955 466 10730 29981 165 12142 16042 7122 36891 32693 22399 20114 28690 417 3884 39596 29012 37319 21434 6688 27908 29894 25396 29028 19523 19388 27233 15861 13600 40328 49991 40632 5086 32762 21325 33292 4986 30021 35061 4307 29923 3863 36270 16222 27177 17375 5243 12895 41757 5563 19479 32418 45303 30365 7984 20138 21083 11941 21511 42274 12480 47131 13814 10642 2548 38319 13241 40826 47659 47061 
198
51401 29691 23560 29339 16275 87603 46414 75242 35195 38090 67485 54242 75264 26379 23010 75768 41940 83184 61466 75896 1668 21235 56636 20784 75305 86730 20970 70031 37 11068 9279 36976 60419 37717 9151 25803 40850 97302 94813 85786 49665 25838 49355 3623 78110 20270 28424 8719 21740 71647 68756 39271 84510 23798 80305 26364 57990 9652 26787 94243 76520 26526 45317 85395 48242 46173 52004 79121 35322 29375 33599 81935 86429 42467 87372 61064 30201 83511 91514 72419 17609 43157 89338 7382 72279 32255 30083 18858 66376 52619 33575 19090 1742 72314 65247 45028 16627 70172 48506 64342 96984 71336 82419 17882 40007 56330 93354 92684 52699 61914 23010 81058 44559 84859 22734 27333 77220 45679 32134 27888 3180 93860 61347 79383 34451 5819 53529 67670 87102 70590 75306 79637 73981 35512 49600 34809 55780 53851 64423 15459 45310 81558 43295 24016 2906 67525 89921 5470 53978 46765 66820 97772 84137 57403 54929 69191 22072 92388 43800 18112 930 65901 56499 65592 1447 42185 82498 85229 87063 70618 19138 57686 94729 57168 32011 12123 35373 14444 92118 25042 11418 70731 61999 95959 77513 87014 1567 30061 93946 89405 54971 15380 43493 49767 41027 20375 96445 319 
3601 2371 20563 34897 42138 10729 34810 30172 21719 24182 32615 40397 6189 16836 40974 43691 29636 38922 32785 19301 18615 43746 12957 20113 25542 41502 37536 11588 14642 47276 11651 31187 42693 19683 6736 40420 20782 35072 38269 2204 46031 9621 23284 35304 43468 44537 29123 17550 33193 2017 5155 44096 21533 12477 11825 1684 42085 44646 18936 46528 35432 48273 38786 20252 19167 34288 15572 49637 27276 32036 35758 17255 34632 22382 24965 33685 39145 14026 13620 31976 16554 38350 31054 23889 13005 11172 38362 10698 28439 3129 25661 33833 2540 5291 16608 3616 26834 34992 5346 16518 45111 9733 42470 34489 30996 11653 34599 4707 43559 31385 38644 5096 32159 2957 30502 41631 41336 33106 9825 25948 33588 37190 22584 2672 10817 29924 29274 42503 26758 2692 32666 49548 42219 15256 17060 39627 16154 3462 39755 12704 30540 24715 4909 31599 26234 16715 39035 37849 6552 4345 18665 46270 15844 40784 9017 48055 48907 33735 28197 46412 8264 23081 7505 5813 36556 533 21603 35599 19463 43883 32561 23727 3542 17330 45877 1660 26384 48779 43909 40680 5539 21287 32468 8386 21075 32729 41903 19496 24076 42434 19579 6214 21370 44289 16121 31382 47473 9539 
196
80129 1098 49012 98611 10552 7512 64742 40734 49007 78132 21483 16058 2809 4494 99033 72783 84536 20403 88430 87320 67554 13790 29795 57262 72393 67013 70136 35285 32184 52555 92515 95815 31779 48769 54335 80519 98971 30254 60847 43765 99775 12031 9945 68471 99987 198 3251 80242 98078 44256 76471 19364 91559 9461 39617 16864 64331 56641 87852 50359 25117 27773 56670 54344 27308 37385 91263 94345 22364 62950 95701 76971 84491 10008 65913 61641 11696 84745 48718 59582 88104 55500 48574 50348 60438 43644 98171 8291 42945 58840 80103 39111 60905 24121 21566 57368 44462 63937 68808 10994 73542 34043 27829 78839 78244 42062 38629 70571 97002 94039 92982 91214 78382 62683 53021 24904 80237 89107 41090 79927 60432 82878 99268 22568 8504 87381 26936 57349 68924 79615 12347 90040 96133 95079 1049 78223 46483 27337 90349 21101 38096 85418 90710 99653 26644 18450 26760 14180 63275 7285 20388 98192 88739 56109 19160 5362 91057 45257 32251 23142 80117 45262 60285 66237 17394 79558 11713 48074 64295 61860 99706 29425 33567 58640 2831 35921 18815 27989 18074 30027 15599 69270 70002 82078 11149 56714 90481 1763 49808 45214 6650 43358 62045 39796 43070 35305 
24197 31085 32366 8383 35900 17015 9398 24018 12796 44509 1474 18164 18808 34425 19059 49018 46033 183 3084 44269 10331 4432 35449 8580 30240 8605 19328 15510 41679 3521 39289 481 24999 3687 20521 4 37016 3761 36304 4928 34594 6314 37149 48657 26588 22757 8630 13173 353 19416 31256 21025 6343 4657 17223 35481 18358 40104 47523 801 10681 9859 28375 11417 48959 44836 22512 20358 27655 44653 22408 31279 45818 39690 2657 21365 7932 21886 32569 37457 27290 10841 14271 1051 16161 2364 5405 31653 32596 49617 8959 24573 2351 29311 47322 29410 41512 30113 12391 24393 6023 26003 18002 31441 14679 27511 8991 40876 10686 32969 28383 24716 41573 5274 9975 41691 2805 40748 26146 41703 21511 2064 23261 42110 27650 7826 36034 39404 36739 44151 29768 24963 34139 36888 39213 17951 35947 35398 216 8683 2740 1283 46028 45057 46963 21092 14858 15846 40039 13026 25921 10208 10550 21695 32070 24532 35143 19334 12394 24618 3212 45301 48694 6666 44674 13620 31515 11251 27831 10340 45208 41793 47476 32803 37128 2746 28541 3226 47484 12207 22229 10697 13647 19359 48271 42082 28426 17825 6139 34948 17728 16879 22224 15078 21195 35475 
198
21233 46749 84134 95209 78927 55373 2348 72969 42003 87033 66518 31085 26889 65795 31634 27062 55020 28605 86200 49287 51749 11391 87462 14630 95193 38807 18765 77285 77069 25242 5243 21030 2613 2175 48303 10768 30251 51717 39369 77057 98935 24536 47148 55588 17247 2612 32835 81615 45201 66515 47219 51107 98963 99106 23027 52778 92212 23251 71798 23942 83504 48117 37397 9257 79558 51212 11948 47479 7666 36286 5979 74496 39721 13432 86021 54422 63660 44202 67170 15198 477 83640 96745 63511 16688 7590 97873 33650 94066 40240 85710 97131 31730 67700 86474 21439 52821 20021 4691 39145 86042 94645 9263 91261 23432 74327 67679 49098 59771 99967 26067 72581 65959 17713 39811 61267 89673 68417 92813 29245 25134 53025 46503 23194 97577 8326 89923 51729 67271 88966 75909 1498 27229 68241 36551 5851 17109 71927 27542 79217 53460 30162 22405 25044 65289 21149 30613 6130 17354 29548 73982 26257 69073 32828 97774 87741 63131 33392 99890 71165 27041 54817 20759 70042 6083 80467 65538 11477 74762 90710 14009 66287 45129 45888 14828 50123 38519 35045 74203 69507 55073 51963 53284 58860 86409 39540 33962 17113 38669 63870 88290 5004 78680 61791 86937 62187 86139 3858 
29399 38108 46352 30481 39177 49366 25549 40593 27990 18283 37981 23970 17146 29023 37833 37816 2862 15557 20699 29199 6103 17258 12160 44403 10388 43123 47865 3451 17141 5532 17058 49484 31184 36540 8092 39463 10771 15539 30879 11808 46995 49931 14231 46456 15167 10709 4467 31436 30042 16998 44792 32222 32083 8822 4290 24208 34023 25498 17324 48597 5098 17298 21421 35226 8954 36951 1901 32568 38240 1722 30479 49864 16344 26927 8059 14446 17052 32137 8254 8616 16744 9904 23993 21435 33686 14023 10968 3622 39131 21342 35989 23065 40795 41067 5636 3512 4286 2113 40759 5665 13865 1020 45915 19959 28979 8592 27571 39007 38166 34375 45258 18359 42829 20164 48843 32406 8710 26664 9269 15774 11417 22018 36128 24782 10640 37535 2629 5740 40134 8942 45548 20181 26790 31397 17379 49690 2699 38406 22963 6345 34548 22584 2217 42735 24340 33318 23688 27539 4927 4686 7317 22052 27313 15522 31056 42072 36905 18794 6070 48145 20820 1278 45060 18868 13065 1491 2321 12186 22515 33621 30296 22713 2631 23719 25197 29377 11485 42488 15601 39283 5965 43309 47724 770 11802 41521 34390 2511 20118 26613 31653 43170 14483 46685 33647 13153 5648 10615 
196
76303 57895 14593 77072 52757 38306 52430 55119 97026 23102 11294 95760 45669 68510 31535 28235 33420 47736 47899 41233 86046 70498 51633 49476 48323 24461 4622 52656 36402 40759 70179 54806 65550 30150 911 60803 77937 2991 79392 62759 87005 37925 10858 82522 72230 21847 85543 7459 33389 18353 92115 5047 53989 5800 68935 49943 9995 77532 60130 3146 3612 9100 35513 9464 7919 28882 91021 4137 81139 1884 71752 64732 50551 69059 94099 25992 72771 11604 99629 3770 55264 29525 14785 14679 54228 19204 93717 13308 12705 12202 10491 52826 30531 74517 72288 97606 77861 93088 71115 21428 6095 94104 52218 67819 64532 52861 82408 82465 93441 94261 98916 22971 83389 64593 10300 1114 71459 36821 86268 18405 75967 43620 11305 2331 9314 70580 79339 72754 71275 68038 73183 55236 47378 43345 20363 24567 89424 45440 44625 26305 62299 46046 57670 28465 15103 11471 49930 8352 72302 42357 59419 63802 40742 99943 48046 30846 72854 43888 3001 32519 55873 63342 12822 85464 48718 77658 62749 74527 71006 88212 3825 13808 94408 57887 97638 57143 67519 81209 57454 31250 44832 96372 71780 61100 57628 99696 46923 4541 82237 62479 42127 32105 93091 91487 61531 85758 
40090 163 6582 9011 43781 47429 45924 26119 29591 33172 30337 47294 24349 11457 24497 30628 45055 25807 41957 45991 22108 30465 36707 23229 20636 15735 13409 40959 23771 15835 21409 17814 20152 41485 5978 4435 11651 3941 11615 9127 37785 42 25780 10232 49389 27575 43833 45563 4308 11723 3130 37889 32801 26277 46631 28832 49123 10814 28319 20916 44543 8752 18257 30808 7514 16625 49302 41646 11145 47669 29420 27378 4794 718 26403 10547 6352 18317 15296 9103 49724 1772 12771 43790 13966 16886 34222 33953 22846 5180 16744 818 34350 25931 42467 1852 48488 34504 10808 16133 30649 27879 23550 48063 40687 17223 23331 38553 9801 20177 46927 22546 22073 43152 13066 44346 21922 40122 38557 11297 46388 31556 25325 40611 2267 28306 49988 175 1825 29177 25776 25060 16131 3451 49317 13516 10086 48789 2903 26937 5390 10323 15719 46517 5972 14381 15155 21883 28497 39744 6291 36664 884 24334 723 11258 22878 24811 46463 30915 45189 18863 35695 42151 43560 20539 4399 20583 38596 3969 48644 6991 10798 4897 9364 33731 45313 48544 28457 21171 10357 31536 46851 4804 1633 47582 25465 47869 25605 4334 29911 10179 37330 35922 28960 17685 
198
11378 64113 96612 70806 20954 56173 15239 58695 31420 13373 27766 18449 4189 12484 41322 61687 70849 46591 10408 5170 25046 72779 31017 35488 54121 70264 18245 29488 97999 29085 77537 35575 21801 49136 21536 38471 27488 69869 84016 4153 60688 48117 51637 2428 89351 18610 10455 7055 35616 60742 49696 13967 57600 62203 98847 72267 12114 87217 91703 5555 85846 61822 85632 92351 38568 69608 33793 33409 70196 57007 38627 31915 37771 64881 45752 34569 61783 4223 92168 62807 28658 74971 20152 19215 60740 41441 20141 68169 68174 95495 2483 41974 90910 67894 38384 77662 58785 8041 69238 83378 10665 72500 7099 48895 54255 64956 64758 68164 5230 46408 40018 87420 86368 79983 96898 56866 2551 13125 37864 66015 17005 2935 47894 5291 68982 70409 63098 58183 61518 65020 17941 99744 85655 28860 37229 38862 6285 5561 50287 67835 78619 9252 53362 88529 1046 81183 7605 49845 34726 91209 73540 671 36939 73243 12212 61571 57782 39954 3456 87572 81702 50766 48526 8565 55051 13785 62084 77644 18216 82945 25915 49047 95097 7666 85758 26291 45232 3726 1369 97960 20292 78325 30609 51088 55450 15915 27208 8442 11419 29042 17851 63571 96705 34777 77090 60735 96819 74235 
29419 15672 18766 43701 8922 1921 1796 7140 44412 23573 16388 36971 44090 9257 18036 49346 8265 21216 9897 49020 32880 30216 29768 36415 5054 39416 36952 32040 43476 38812 32709 12491 13622 49494 18223 29051 41932 13636 44250 13594 10337 446 36739 27641 38364 9269 17532 18904 23546 20332 21656 13106 29039 37264 10837 42238 10889 32864 21419 30504 8100 32738 18946 35418 17146 11814 25392 16732 37884 46136 47570 5902 48763 38357 39806 49130 37565 24490 44100 18501 10742 654 15628 19617 44877 27737 30023 39870 3219 6247 46553 47139 9901 28357 10185 10529 24493 15594 35822 3499 44379 17853 25220 17358 31107 24765 17685 22689 43440 13385 6050 23163 17468 38540 47687 45655 16269 45323 1484 13303 27904 19772 27190 13524 9258 19297 24734 27715 35673 18957 10897 29896 25943 45218 46870 20231 44627 31574 43845 49528 31181 6165 19601 26872 44761 49040 12684 49693 30491 2940 3520 40320 3966 42921 34382 26751 41688 45768 40556 5071 718 48980 13402 16094 23942 16684 41518 5750 24853 40009 36913 47783 10468 8181 24411 4655 23000 42696 49264 30880 49953 16324 1987 17658 44498 18664 1619 32516 29985 9327 15239 26253 10341 46552 20255 8872 28068 8313 
196
30144 63488 3464 68981 37020 39355 53390 87557 42872 31076 15500 54112 49083 42237 22406 54328 24092 65434 15144 76476 89688 73259 26901 61617 66037 46212 246 62922 12418 94536 980 79686 63575 27092 83773 57657 52633 93202 83136 96759 27941 54614 40282 92374 44613 54880 18167 80704 86246 85255 7152 62159 36940 93350 49946 56637 19582 47137 60484 23301 93718 92948 14196 1367 67547 55431 45688 72578 94824 81084 48854 68947 5887 37934 51704 8964 47216 27777 57434 94286 1398 12609 84520 12887 85200 90461 24455 33859 96294 56054 98466 37307 73280 1705 73767 41638 76609 43261 35259 66038 45955 98127 16713 87857 8617 83252 37755 63640 43368 34122 46621 56877 5538 88277 70403 58051 88895 71274 23115 28099 70720 14478 6984 7341 11311 98820 90921 63740 98797 36178 18305 8389 35883 55829 2046 93121 20540 36899 97727 25789 35922 87832 6836 10512 52724 95439 37119 10660 86062 22715 38347 87188 66555 45834 62455 45455 86093 73621 27731 22980 98051 25560 8090 51913 46970 52677 36556 9677 4650 63266 14546 24083 75208 51967 6667 49780 82323 71513 6130 96729 43 15323 6692 37502 61728 24389 23220 54049 23366 35207 26518 18748 85784 22555 65374 34207 
2102 37261 11734 37310 7536 29071 14963 24004 5623 44649 4245 8584 10096 22601 27358 34918 28341 13568 48396 30373 25713 16968 27330 28454 43039 23612 10855 3117 5394 16972 6460 35914 40301 23017 30211 24468 35837 12223 47606 29971 32942 20400 12146 14916 21859 22040 2739 3706 13321 18834 42890 38371 43667 38836 47489 17767 26376 4235 35764 38795 1979 45058 28715 39739 21032 8003 34246 19799 40838 28012 14510 44468 5724 38843 11021 20561 33422 14317 4529 28883 35639 17176 17927 43720 31394 2014 34470 22709 9266 30706 43500 34691 36609 38452 34757 6729 21127 7830 5846 45115 2141 38611 18019 23449 12858 41811 18400 35971 44375 17480 47126 9529 40056 29716 306 9628 44361 19375 41312 38936 14063 7970 5405 14372 34733 8295 6912 29065 48463 39503 41158 43970 40026 23451 24283 29083 32208 44249 38001 33991 48103 7481 19784 30918 7191 38321 39989 697 8355 10326 24141 18588 6908 45767 18293 17244 37316 14007 8603 25480 9518 24866 47353 2013 1326 16755 18409 48396 35429 42805 4586 6141 20731 43067 5553 17294 6642 37530 32424 39436 20911 11205 39114 19741 855 5277 17622 16873 32230 16727 9736 42967 12172 32622 3344 14498 
196
60588 7297 63069 8211 84667 45213 85614 60834 56923 27893 83394 89908 8302 39852 89520 40353 35547 30597 5105 59816 42918 4476 26894 75584 43903 50248 57524 98797 16147 27344 60588 59925 71896 13755 64906 17792 91697 31557 83191 89037 44201 37458 84996 51822 78499 87593 81640 14576 32775 3497 49268 65730 97005 48476 27071 91885 55874 96699 60287 28925 12649 52211 58702 9408 85578 34680 95784 11012 86239 63688 98025 38102 86290 16535 53966 68311 33436 35898 18461 83451 8748 51260 54247 27704 32267 61977 82227 12234 74821 34142 1826 74845 96107 62441 46106 33976 73183 45802 1884 49810 10561 7825 60498 27676 37383 95794 91424 52873 66945 73160 9654 90490 69237 54433 72725 79712 4232 48785 27578 31723 94206 29152 94824 64988 42531 85692 9596 47496 48661 99717 85308 12500 95617 13798 72836 62711 35301 15551 14883 38778 63233 33532 53586 95584 8678 9954 80529 41047 12521 71982 7995 96251 92809 4215 99451 32384 73702 29868 17893 86965 14646 68794 87175 55977 16441 46131 96526 31764 35918 1417 58250 72308 53956 36495 27157 4402 3438 31155 95894 78466 45427 4288 27610 68122 38081 89760 88880 7164 62254 94919 26688 97338 85107 41774 76802 60237 
16766 22098 21322 36335 27682 11023 42062 9358 38032 18308 47619 37514 37076 15898 19293 46353 25645 42571 48867 8326 31071 27402 28199 47195 15815 533 34501 11282 4421 3549 39277 13104 36278 22943 8461 36357 27781 4023 27927 3868 13430 37056 26259 22112 10614 23618 43130 29218 1772 44505 20004 26022 45164 28427 11561 20565 31750 48945 46073 44078 13513 31400 1357 21289 19072 46058 3722 15341 49873 6434 43564 44031 6138 45573 32985 41560 24453 8077 5144 40681 5774 39224 48823 12011 33753 36986 37025 47202 42661 36274 7714 43801 20601 42396 23954 8433 32948 6971 18304 43682 3113 35885 42945 13337 7037 5935 17612 1745 12247 25520 16578 37312 40615 25719 23798 39081 36341 15170 29606 19173 4291 8059 26169 9435 24151 7594 15884 28688 3614 27421 44497 1374 32518 27211 4900 17301 16334 29915 18977 21285 18677 35280 18623 49125 17162 14009 16153 37925 6571 6446 23651 19274 45779 20867 27137 8596 19541 4588 37690 8277 12987 21237 31082 35346 39311 21459 16172 15947 17308 29587 43128 17115 33580 15650 20821 35808 25962 43787 22297 43749 13598 43254 11410 13041 7329 3004 8060 41964 27286 49622 15752 24820 39891 47740 32770 19081 
199
10168 89012 63418 39668 10473 55389 38332 20469 12499 33493 54926 12343 15320 6737 90477 3089 36113 13183 96645 72343 93244 59159 72767 36167 35360 54572 29790 42001 63724 83353 23070 29826 7328 88169 24495 63035 84513 81860 79392 94424 19056 10914 72602 80883 8266 76797 88540 55947 90181 25209 13178 42527 8051 95854 5655 47133 23541 11445 47969 5172 87741 99535 47448 22851 41569 60306 66911 29009 79829 96577 4250 87360 92559 53702 27428 63256 23032 95526 1728 75246 60995 55892 55630 2774 60114 37007 55776 13149 60382 63338 84379 10232 51620 35185 51822 33448 10783 63030 51138 62613 65261 37761 77130 3219 7270 89098 52894 14611 30888 77535 34493 36258 28615 98741 63265 46781 26484 34095 90469 48032 31721 7859 46251 86062 34814 78645 81621 51824 35980 66977 26337 67435 96652 97836 14756 72453 17722 88473 67790 47678 58992 50430 66469 35688 48276 1502 21146 88011 72880 17142 1906 85368 268 62388 91647 24194 41605 29574 60806 77820 23490 26392 52983 4641 75623 15036 43526 678 32628 84127 98864 19301 13063 22083 3801 52185 68952 59386 68541 92450 43569 3160 62491 75119 15716 31288 98393 20742 53124 79690 342 68873 18668 46359 9623 7212 26929 17932 14629 
49203 19448 37834 30613 18903 39345 37648 42778 15838 40039 37419 36613 38496 23990 49653 43109 24869 36051 2543 49894 7189 38829 47759 24570 33524 29519 29991 39720 18336 5235 8188 48690 37165 41874 36801 26775 25848 42909 3239 27682 27926 45533 5341 22060 41781 32568 41217 23826 29119 38168 20230 35626 11773 30819 16196 41434 47735 14649 35283 32650 32426 4334 34125 16927 40964 9917 12072 12525 14268 5951 29767 38856 5467 44108 5484 7051 44933 16503 19153 9880 4665 30100 29188 19103 29513 3733 26061 7530 21516 19786 3278 34659 21534 15729 45512 41425 11529 47498 29104 13167 42681 38235 13737 10867 3567 40671 34340 19646 29294 41429 43726 44828 16971 17813 17615 1492 1022 29030 46606 22289 15483 19184 26354 13785 47682 38519 14357 38484 46419 12974 28488 11915 28646 32264 23681 19003 33537 35340 28745 44396 30997 47793 17288 23944 30105 36947 28222 37053 10372 43727 29826 8578 5761 48123 15788 27045 6997 18661 46833 41333 21396 27150 44333 32654 18264 25755 35757 19598 16016 14089 14843 42129 28268 13647 20456 35972 14345 10698 36791 19298 18195 25516 4274 42360 4386 23254 34306 22126 23919 39723 8710 41434 34020 37387 23465 46181 46330 19144 34030 
