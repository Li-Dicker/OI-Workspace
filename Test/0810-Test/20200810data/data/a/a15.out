Case #1: 13798571
Case #2: 14490282
Case #3: 14492919
Case #4: 14031031
Case #5: 13802713
Case #6: 13890242
Case #7: 13837412
Case #8: 13615845
Case #9: 14206863
Case #10: 14236742
